df.buildings.xml

```diff
@@ -459,20 +456,20 @@
 
             <vmethod name='categorize' comment='Add to world.buildings.other.*'> <bool name='in_play'/> </vmethod>
             <vmethod name='uncategorize' comment='Remove from world.buildings.other.*'/>
-            <vmethod ret-type='int32_t' name='getBaseValue'/>
+            <vmethod ret-type='int32_t' name='getArchValue'/>
             <vmethod name='setTriggerState'> <int32_t name='new_state' comment='0 = active/open, 1 = inactive/closed'/> </vmethod>
 
             -- 74
 
             <vmethod ret-type='bool' name='needsMagma'/>
             <vmethod name='removeUses'> <bool name='noscatter'/> <bool name='lost'/> </vmethod>
-            <vmethod name='deconstructItems'> <bool name='noscatter'/> <bool name='lost'/> </vmethod>
+            <vmethod name='deconstructItems'> <bool name='noscatter'/> <bool name='lost'/> <bool name='from_damage'/> </vmethod>
```

Comment:
could this be why we have crashed in the past while trying to forcibly deconstruct buildings? This was a key issue what I was trying to write `quickfort` regression tests since I couldn't test constructed buildings (since I couldn't clean those kinds of tests up afterwards)

Reply:
I suppose it's possible, though I don't recall whether any of our currently supported platforms still pass parameters on the stack and expect callee-cleanup (which is the only case where I'd expect something like this to cause a crash).

Reply:
would have caused stack alignment issues on msvc but not on gcc